lib/happo/public/happo-runner.js

```diff
@@ -162,8 +162,27 @@ window.happo = {
     }
   },
 
+  // Scrollbars inside of elements may cause spurious visual diffs. To avoid
+  // this issue, we can hide them automatically by styling the overflow to be
+  // hidden.
+  removeScrollbars: function removeScrollbars(node) {
+    if (
+      node.scrollHeight !== node.clientHeight
+      || node.scrollWidth !== node.clientWidth
+    ) {
+      // We style this via node.style.cssText so that we can override any styles
+      // that might already be `!important`.
+      // eslint-disable-next-line no-param-reassign
+      node.style.cssText += 'overflow: hidden !important';
```

Comment:
I thought maybe, but I tried this out and it seemed to work just fine like this.